test/atomicrmw.ll

```diff
@@ -26,37 +25,37 @@ target triple = "spir64"
 define dso_local spir_func void @test_atomicrmw() local_unnamed_addr #0 {
 entry:
   %0 = atomicrmw xchg ptr addrspace(1) @ui, i32 42 acq_rel
-; CHECK: AtomicExchange [[Int]] {{[0-9]+}} [[Pointer]] [[Scope_Device]] [[MemSem_AcquireRelease]] [[Value]]
+; CHECK: AtomicExchange [[Int]] {{[0-9]+}} [[Pointer]] {{.+}} [[MemSem_AcquireRelease]] [[Value]]
```

Comment:
And here is an opposite situation?

Reply:
Yes, just to have at least one test where Relaxed explicitly shows up. I can flip it to pattern match like the other test, if you think that is preferable.

Reply:
I don't have strong objections against the current version of the tests